pkl-core/src/main/java/org/pkl/core/stdlib/protobuf/RendererNodes.java

```diff
@@ -522,6 +523,14 @@ public void visitDataSize(VmDataSize value) {
           .build();
     }
 
+    @Override
+    public void visitBytes(VmBytes value) {
+      throw new VmExceptionBuilder()
+          .evalError("cannotRenderTypeAddConverter", "Bytes", "Protobuf")
+          .withProgramValue("Value", value)
+          .build();
```

Comment:
Other renderers use `RendererException` in these cases. Worth making these consistent?

Reply:
This is an in-language renderer, so `VmException` is correct here